docs/getting-started.mdx

```diff
@@ -0,0 +1,237 @@
+import HowItWorksDiagram from '../assets/getting-started-how-superface-works.svg';
+
+# Getting started
+
+Superface is all about you, the consumer of API capabilities. To get started, first pick a capability and its provider, install it to your App and use Superface OneSDK to make the call.
+
+## How Superface works
+
+Unlike every other API client, absolutely **no API information is hard-coded in your App or the SDK**.
+
+The instructions how to make a call are downloaded during your App runtime and resolved by OneSDK to make a **direct HTTP** call to the API. The calls go straight from your App to the provider's API and are **never proxied through Superface**. This lets you change the instructions at runtime without updating or redeploying your App. You can even change providers while it’s running.
+
+<figure>
+  <HowItWorksDiagram />
+  <figcaption>
+    During the app development you install a capability and OneSDK. In the
+    runtime the instructions for calling the API are downloaded from the
+    Superface registry and the API is called directly from your app.
+  </figcaption>
+</figure>
+
+## Superface in 3 minutes
+
+The following steps will guide you how to quickly get started with Superface and OneSDK.
+
+:::info Prerequisites
+
+This guide assumes you have [Node.js](https://nodejs.dev/) version 14 or newer installed.
+
+:::
+
+<div class="count-steps-h3">
+
+### Pick a capability
+
+Decide what capability you would like to use in your App. A capability is described by its profile with unique ID.
+
+In this guide we will use the capability **List user repositories of a VCS** like GitHub or GitLab. Its ID is `vcs/user-repos`.
+
+:::note Capability details
+
+To see a detailed description of the capability, check the [`vcs/user-repos` profile page](https://superface.ai/vcs/user-repos).
+
+You can discover more capabilities at [superface.ai](https://superface.ai/catalog).
```

Comment:
Catalog is locked and this doc is public. Do you think it might be worth mentioning you need an account for browsing the curated catalog?

Reply:
Oh, that's good point since you don't necessarily have an account. I will create a Prerequisites section and mention it there.